static/js/place/menu.tsx

```diff
@@ -139,7 +139,7 @@ class Menu extends React.Component<MenuPropsType> {
           </li>
         )}
         {categories.map((category: string) => {
-          let items: string[][] = [];
+          let items: string[] = [];
```

Comment:
I changed this because I found the type of "categories" wrong when I tried to pass props to "Menu" in the test. 
So I changed the "categories" type to { [key: string]:string }, and modified the type of "items ". 
I was also surprised that typescript allows list to take string as an attribute or index in line 152.  But I kind of hesitate to change it from list to object. For line 144 to 147, it creates a list. I'm not sure if there will be any side effect if I change that to an object.

Reply:
oh, yes, the categories change makes sense and thanks for catching that bug!

But for the `items` part, if you look at line 152, we're saying that at that index of `items`, we are adding an array, which means `items` will hold multiple arrays, so it doesnt make sense for the type here to be a single array of strings?

I think it will be cleaner to have items be an object, and make the changes accordingly. For line 144-147, we can just set `items[""] = <the array>`

Reply:
Got you! Done.